cypress/integration/e2e/table/schedule-modes.spec.ts

```diff
@@ -25,6 +25,17 @@ describe("Schedule modes spec", () => {
     });
   });
 
+  context("not saved changes are not visible", () => {
+    it("performs test", () => {
```

Comment:
Zamiast `"performs test"` 
`context("when changes are not saved", ...`
`it("does show the changes",...`

Reply:
Ok